docker/sp/pom.xml

```diff
@@ -157,6 +157,26 @@
             <groupId>io.cellery.observability</groupId>
             <artifactId>io.cellery.observability.k8s.api.server.client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.oltu.oauth2</groupId>
+            <artifactId>org.apache.oltu.oauth2.client</artifactId>
+            <version>1.0.0</version>
```

Comment:
Include the dependency versions to the root pom.xml, and the infer the dependency in the child modules.